src/fsharp/NicePrint.fs

```diff
@@ -892,7 +892,7 @@ module private PrintTypes =
                 | tys -> bracketL (layoutTypesWithInfoAndPrec denv env 2 (wordL (tagKeyword "or")) tys)
             tysL ^^ wordL (tagPunctuation ":") ---
                 bracketL (stat ++ wordL (tagMember nm) ^^ wordL (tagPunctuation ":") ---
-                        ((layoutTypesWithInfoAndPrec denv env 2 (wordL (tagPunctuation "*")) argtys --- wordL (tagPunctuation "->")) --- (layoutTypeWithInfo denv env rty)))
```

Comment:
Could probably be simplified to `layoutReturnType`

Reply:
Fixed in https://github.com/dotnet/fsharp/pull/9728/commits/6c2b3e67d1cf9f0f3b39572940d8e0293d750469 by changing `layoutReturnType` to be in-scope for this line.